src/components/pipeline/PipelineForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import Grid from '@material-ui/core/Grid';
+import SaveIcon from '@material-ui/icons/Save';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+
+import {
+  EDIT_PIPELINE_SAVE_BUTTON_ID,
+  EDIT_PIPELINE_NAME_ID,
+  EDIT_PIPELINE_DESCRIPTION_ID,
+} from '../../config/selectors';
+
+import PipelineAccordion from './PipelineAccordion';
+
+const PipelineForm = ({
+  pipelineAlgorithms,
+  algorithms,
```

Comment:
Let's also discuss during the call:

- maybe for extra clarity we can import this as `applicationAlgorithms`
- it seems this is received as a prop then passed as a prop to `PipelineAccordion`, but is no longer used in `PipelineAccordion` (let's discuss - I might be missing something)